app/controllers/api/v1/websites_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def create
     @website = Website.create(domain: website_params[:domain], title: website_params[:title], member_id: current_member.id)
+    @subscriber = Subscriber.create(name: current_member.name, email: current_member.email, website_id: @website.id, role: 1)
```

Comment:
automat ca sa mearga `@website.members` trebuie sa adaugi in `website.rb` `has_many :members, through: :website_members`